packages/vue-inertia/src/index.ts

```diff
@@ -1,11 +1,42 @@
-import { Config, NamedInputEvent, RequestMethod, SimpleValidationErrors, toSimpleValidationErrors, ValidationConfig, ValidationErrors, resolveUrl, resolveMethod } from 'laravel-precognition'
+import {
+    Config,
+    NamedInputEvent,
+    RequestMethod,
+    SimpleValidationErrors,
+    toSimpleValidationErrors,
+    ValidationConfig,
+    ValidationErrors,
+    resolveUrl,
+    resolveMethod,
+    Validator,
+} from 'laravel-precognition'
 import { useForm as usePrecognitiveForm, client } from 'laravel-precognition-vue'
-import { useForm as useInertiaForm } from '@inertiajs/vue3'
+import { InertiaForm, useForm as useInertiaForm } from '@inertiajs/vue3'
 import { watchEffect } from 'vue'
 
 export { client }
 
-export const useForm = <Data extends Record<string, unknown>>(method: RequestMethod|(() => RequestMethod), url: string|(() => string), inputs: Data, config: ValidationConfig = {}): any => {
+type Precognitive<Data extends object> = {
```

Comment:
Not 100% sold on this name for the type, but it is private. 

Considered joining with a `laravel-precognition` type but this is probably more resilient since it's not a true union of attributes.